src/i18n/locales/pt.json

```diff
@@ -165,82 +165,82 @@
       "loadMoreTransactions": "Carregar mais transações",
       "viewTransactions": "Ver transações",
       "notOpenedYet": "Esta conta ainda não foi aberta",
-      "notOpenedYetReason": "Embora o endereço da conta seja válido, nenhum bloqueio foi publicado em sua cadeia ainda. Se NANO foi enviado para esta conta, ainda precisa publicar um bloco correspondente para embolsar os fundos.",
+      "notOpenedYetReason": "Embora o endereço da conta seja válido, nenhum bloco foi publicado em sua blockchain ainda. Se NANO foi enviado para esta conta, ainda precisa publicar um bloco correspondente para embolsar os fundos.",
       "missingAccount": "Conta faltando",
-      "missingAccountInfo": "A conta não está especificada no URL. Use a barra de pesquisa superior para pesquisar uma conta ou anexe-a no URL após /conta/"
+      "missingAccountInfo": "A conta não está especificada no URL. Use a barra de pesquisa superior para pesquisar uma conta ou anexe-a no URL após /account/"
```

Comment:
Ahh, I'll have to double check other languages for this one, good catch

Reply:
Yes, I even tried to check if this kind of URL worked, but then I realized it was a translation error.